web/src/components/feature/CalculateFluid.tsx

```diff
@@ -67,6 +74,24 @@ export const CalculateFluid = ({
             setPressure(Number(event.target.value))
           }
         />
+        <TextField
+          id="feed-flow-input"
+          value={feedFlow.toString()}
+          label="Feed Flow"
+          unit={feedUnit}
+          type="number"
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+            setFeedFlow(Number(event.target.value))
+          }
+        />
+        <Switch
```

Comment:
According to the figma sketches. This component should be placed with the "PhaseTable".
I see no reason to change that.

Reply:
Did we conclude to await meeting on this one?

Reply:
Concluded we do as in the sketches, and ask for feedback before moving the component somewhere else.

Reply:
Will fix